.github/workflows/main.yml

```diff
@@ -1,10 +1,13 @@
 name: Build dss-ecosystem
 on:
-  push:
```

Comment:
Why is `push` removed?

Reply:
Push is removed by considering the below-mentioned scenarios.
1. Assuming that we won’t be directly pushing to the master or stable/* branches.
2. If we add push then on merge also it will be considered as the push to the specific branch and will lead to a re-trigger of the code build projects.
3. As of now, we have added the scenario where the pull request to master & stable will trigger code build and the type of pr will be [opened, synchronize, reopened].
4. When the PR from any branch to stable will be opened, since the base branch is stable/*, the specific ecr image will be used during code build execution & the same goes for the master branch.